docs/reference/docs/delete-by-query.asciidoc

```diff
@@ -384,7 +384,7 @@ POST _tasks/r1A2WoRbTwKZ516z6NEs5A:36619/_cancel
 The task ID can be found using the <<tasks,tasks API>>.
 
 Cancellation should happen quickly but might take a few seconds. The task status
-API above will continue to list the task until it is wakes to cancel itself.
+API above will continue to list the task until the task is awaken to cancel itself.
```

Comment:
Perhaps something like the following?

```suggestion
API above will continue to list the task until the task awakens to cancel itself.
```

Reply:
Well, does the task awaken by itself ? And it wasn't clear for me which task is canceled: the `_delete_by_query` task or the `cancel the _delete_by_query` task... Perhaps the text could be explicit to avoid ambiguities.

Reply:
which task is *awaken* \*

Reply:
> Well, does the task awaken by itself?

Technically the task checks periodically to see whether it has been cancelled, and if so, stops executing.

> And it wasn't clear for me which task is canceled: the _delete_by_query task or the cancel the _delete_by_query task... Perhaps the text could be explicit to avoid ambiguities.

The delete-by-query task is the one that stops when cancelled (with the check I mentioned above).

Perhaps we should clarify it to something like

"Cancellation should happen quickly but might take a few seconds. The task static API will continue to list the task until the delete by query task checks that it has been cancelled and terminates itself."

Reply:
> Technically the task checks periodically to see whether it has been cancelled, and if so, stops executing.

Nice! 

> Perhaps we should clarify it to something like
> "Cancellation should happen quickly but might take a few seconds. The task static API will continue to list the task until the delete by query task checks that it has been cancelled and terminates itself."

The task static API also lists the cancel task right ? What about:

"The task static API will continue to list the delete by query task until this task checks that it has been cancelled and terminates itself."

:)

Reply:
That sounds good to me :+1:

Reply:
Alright!